src/popup/router/components/TransactionOverview.vue

```diff
@@ -0,0 +1,282 @@
+        </a>
+        <CopyAddress :value="transaction.sender.address" />
+      </div>
+      <div class="recipient">
+        <span
+          v-if="transaction.aens || transaction.contractCreate"
+          :class="{ aens: transaction.aens }"
+          class="name"
+        >
+          {{ transaction.recipient.label }}
+        </span>
+        <a
+          v-else
+          :href="getExplorerPath(transaction.recipient.address)"
+          target="blank"
+          class="name"
+        >
+          <TruncateMid :str="transaction.recipient.name || transaction.recipient.label" />
+        </a>
+        <CopyAddress v-if="transaction.recipient.address" :value="transaction.recipient.address" />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+import { OBJECT_ID_TX_TYPE } from '@aeternity/aepp-sdk/es/tx/builder/schema';
+import { TX_TYPES, TX_TYPES_MDW } from '../../utils/constants';
+import TriangleRight from '../../../icons/triangle-right.svg?vue-component';
+import ContractIcon from '../../../icons/contract-create.svg?vue-component';
+import AensIcon from '../../../icons/aens.svg?vue-component';
+import Avatar from './Avatar';
+import TruncateMid from './TruncateMid';
+import CopyAddress from './CopyAddress';
+
+export default {
+  name: 'TransactionOverview',
+  components: {
+    Avatar,
+    TruncateMid,
+    CopyAddress,
+    TriangleRight,
+    AensIcon,
+    ContractIcon,
+  },
+  props: {
+    tx: { type: Object },
+    claim: { type: Boolean },
+    event: { type: String },
+    contract: { type: String },
+    connect: { type: Boolean },
+    app: { type: Object },
+  },
+  computed: {
+    ...mapGetters(['account', 'getExplorerPath']),
+    transaction() {
+      if (this.connect)
+        return {
+          sender: {
+            address: this.app.host,
+            name: this.app.name,
+          },
+          recipient: this.currentAccount,
+          title: this.$t('pages.connectConfirm.connecting'),
+          desc: '...',
+        };
+      if (this.claim)
+        return {
+          sender: {
+            address: this.contract,
+            label: this.$t('transaction.overview.contract'),
+          },
+          recipient: this.currentAccount,
+          title: this.$t('transaction.overview.receiveTips'),
+          desc: this.event,
+        };
+      switch (this.txType) {
+        case TX_TYPES.txSign:
+          return {
+            sender: {
+              address: this.tx.senderId,
+              name: this.getDisplayName(this.tx.senderId),
+              label: this.$t('transaction.overview.accountAddress'),
+            },
+            recipient: {
+              address: this.tx.recipientId,
+              name: this.getDisplayName(this.tx.recipientId),
+              label: this.$t('transaction.overview.accountAddress'),
+            },
+            title: this.$t('transaction.type.spendTx'),
+            desc: this.txType,
```

Comment:
please, use `description` instead, firstly I thought that it is about ordering (like asc/desc in SQL)